TodoList/TodoList.Api/Controllers/NodesController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using TodoList.Api.Models;
+
+namespace TodoList.Api.Controllers
+{
+    public class NodesController : ApiController
+    {
+        private NodeModel[] Nodes { get; }
+
+        public NodesController()
+        {
+            Nodes = InitializeData().ToArray();
+        }
+
+        [Route("api/v1/nodes")]
```

Comment:
There is too much duplicity when you define routes. Either define route in `RoutesConfig.cs` or define one route for whole controller, not per method.  (Something like `[Route("api/v{version:apiVersion}/Todos/{id:int?}", Name = "Todos")]`)